10313.xml

```diff
@@ -0,0 +1,241 @@
+        <ObjectID>10313</ObjectID>
+        <ObjectURN>urn:oma:lwm2m:x:10313</ObjectURN><LWM2MVersion>1.0</LWM2MVersion>
+        <ObjectVersion>1.0</ObjectVersion><MultipleInstances>Multiple</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Resources>
+            <Item ID="6000">
+                <Name>Interval Period</Name>
+                <Operations>R</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Mandatory</Mandatory>
+                <Type>Integer</Type>
+                <RangeEnumeration>1..864000</RangeEnumeration>
+                <Units>s</Units>
+                <Description><![CDATA[The Interval Period resource is an Integer value representing the width in seconds of the intervals being managed by this interval data object. This resource is read only and can only be changed by resource 11 (Change Interval Configuration). It is recommended that the Interval Period be set to a devisor of 24 hours (86400 seconds) to provide a consistent interval period. Examples of Interval Period include:-
+30 = Every 30 seconds
+600 = Every 10 minutes
+1800 = Every 30 minutes
+3600 = Hourly
+7200 = Every 2 hours
+14400 = Every 4 hours
+43200 = Every 12 hours
+86400 = Every Day
+172600 = Every Second Day]]></Description>
+            </Item>
+            <Item ID="6001">
+                <Name>Interval Start Offset</Name>
+                <Operations>R</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>Integer</Type>
+                <RangeEnumeration>0..86399</RangeEnumeration>
+                <Units>s</Units>
+                <Description><![CDATA[The Interval Start Offset resource is a read only resource representing the number of seconds past midnight for which the first interval is recorded. If this resource is empty, it is assumed that the intervals are midnight aligned. This can be used to adjust interval boundaries. As an example, an Interval Period of 3600 seconds and an Interval Start time of 300 represents hourly interval data, starting at 00:05.]]></Description>
+            </Item>
+            <Item ID="6002">
+                <Name>Interval UTC Offset</Name>
+                <Operations>R</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>String</Type>
+                <RangeEnumeration></RangeEnumeration>
+                <Units></Units>
+                <Description><![CDATA[The Interval UTC Offset resource is a read only resource representing the time zone offset for this Interval Data instance. If this resource is empty, the application should use the UTC offset provided in the Device [/3/0/14] object instance resource or UTC if not provided. UTC+X [ISO 8601].]]></Description>
```

Comment:
Good to clarify the format of the string

Reply:
Hi, do you mean outside of the description that is given? Which tag in the XML needs clarification?